docs/network-iot/run-an-lns/buy-an-oui.mdx

```diff
@@ -0,0 +1,253 @@
+---
+id: buy-an-oui
+title: Buy An Organizationally Unique Identifier
+pagination_label: Buy An Organizationally Unique Identifier
+sidebar_label: Buy an Organizationally Unique Identifier
+description: Helium Documentation
+image: https://docs.helium.com/img/link-image.png
+slug: /iot/run-an-lns/buy-an-oui
+---
+
+## What is an Organizationally Unique Identifier?
+
+Each LoRaWAN Network Server (LNS) on the Helium Network is managed using an Organizationally Unique
+Identifier (OUI). This registers the LNS with the blockchain and allocates two very important
+routing objects owned and maintained by the OUI operator:
+
+- Up to five **filters**, describing the sets (AppEUI, DevEUI) expected by the LNS
```

Comment:
We mention filters here, but don't explain it later. What should folks know about this? 
cc/ @lytzV

Reply:
huh idk what that means actually